### PR TITLE
fix: update moderate severity security 

### PR DESCRIPTION
**Pull request by Ibrahimmaga**
# Pull Request

## 📖 Description
CVE-2021-23362
moderate severity
Vulnerable versions: < 2.8.9
Patched version: 2.8.9
The npm package hosted-git-info before 3.0.8 are vulnerable to Regular Expression Denial of Service (ReDoS) via regular expression shortcutMatch in the fromUrl function in index.js. The affected regular expression exhibits polynomial worst-case time complexity

ws@^7.4.6:
  version "7.4.6"
GHSA-6fc8-4gx4-v693
moderate severity
Vulnerable versions: >= 7.0.0, < 7.4.6
Patched version: 7.4.6

CVE-2021-23364
moderate severity
Vulnerable versions: >= 4.0.0, < 4.16.5
Patched version: 4.16.5
The package browserslist from 4.0.0 and before 4.16.5 are vulnerable to Regular Expression Denial of Service (ReDoS) during parsing of queries.

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [x] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

**Review by scomea — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -2708,7 +2708,7 @@
     cosmiconfig "^5.1.0"
     dedent "^0.7.0"
     dot-prop "^4.2.0"
-    glob-parent "^5.1.2"
+    glob-parent "^5.0.0"
```
Comment (radium-v) [inline yarn.lock:2711]: Entries related to `glob-parent` and `trim-newlines` will be resolved once #4867 is merged.

Comment (EisenbergEffect) [reply yarn.lock:2711]: @Ibrahimmaga Looks like #4867 is now merged. Can you update and resolve conflicts?

**Review by EisenbergEffect — APPROVED** [review]
Pending update based on John's feedback.

File: yarn.lock (hunk 1 above)
Comment (Ibrahimmaga) [reply yarn.lock:2711]: Yes. I will resolve the conflict on this

**Review by radium-v — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -151,7 +151,9 @@
     "ts-jest": "^25.4.0",
     "ts-node": "^8.8.2",
     "typescript": "^3.9.0",
-    "yargs": "^16.2.0"
+    "yargs": "^16.2.0",
+    "ws": "^7.4.6",
+    "browserslist": "^4.16.5"
```
[the remark this answers is not in the document]
Comment (Ibrahimmaga) [reply package.json:156]: true. this update is being done through a resolution so we don't need it locked there. or are you thinking of a different reason?

[the remark this answers is not in the document]
Comment (radium-v) [reply package.json:156]: That makes sense. In that case, can you move these entries to the `resolutions` array?

**Pull request merged**